### DISCUSSION TITLE
Bug:Firefox: Normal web site like protected by browser randomly.

### DISCUSSION THREAD

**Discussion by yangu-hury** [darkreader/darkreader#8960] [General]
Normal pages randomly get protected by browser or Extension bug. I really don't now who bugged is. Firefox only no other browsers.

![9QjLgSkWml](https://user-images.githubusercontent.com/36553429/170479497-d995696e-4b28-4a20-8d6c-df652b9e910c.png)

**Comment by alexanderby** [comment]
Hi, please share some links when you encounter this again.

Do you use "Invert listed only option"? Have you ever enabled v5 UI preview?

**Reply by yangu-hury** [reply]
It happens with any link I really don't know what make it happens. I use "Inverted listed only". 

It magically fix alone if you wait for sometime like one minute using a another tab than back to the protected tab or reload and fixed.

I get it randomly when I create a new tab search from Firefox home page, open a google result, then page protected. I tried it again and the problem don't happened. I back to this tab I am writing here at github then protected again. Just randomly. I am unable to reproduce.    

![AeYHLMXzJ9](https://user-images.githubusercontent.com/36553429/170890516-09848a99-ef95-4f77-b393-c13327ff3b3e.png)![FPs40M7NNc](https://user-images.githubusercontent.com/36553429/170890520-85117fce-0924-464a-9c05-f18d96941585.png)

**Reply by Gusted** [reply]
Do you use any extension that modify the tab's URL?

**Reply by yangu-hury** [reply]
No. That happens with less frequency today, probably because it was a bug in old Firefoxes. Today I see the dark and light mode switching OK, but the icon V and X before the site name(domain) doesn't change in Dark Reader interface sometimes(random).
